docs/decision-records/DR-0002-dr-validation.md

```diff
@@ -0,0 +1,86 @@
+- (after the decision has been recorded / PR merged) the **decision** is applied, respected and understood (i.e. if someone has a question about this **decision**, they should reach to the **Sponsor**)
+
+The **Sponsor**, and the **Corresponding Author** SHOULD be clearly defined in a markdown header of the **Decision Recording** following the decision framework templates.
+
+A Pull Request needs Reviewers and an Assignee.
+The **Corresponding Author** and the **Sponsor** are not eligible for those roles. Anyone else is eligible.
+
+Anyone MAY, and at least one of the {**Corresponding Author**, **Sponsor**} MUST designate one or more eligible Reviewers.
+An eligible Reviewer MAY designate themselves.
+
+The **Corresponding Author** OR the **Sponsor** MUST assign the **Decision Recording** Pull Request to an eligible Assignee.
+The Assignee MAY also be a Reviewer.
+
+The Assignee looks for consensus before merging the **Decision Recording** Pull Request.
+In clear absence of consensus the Assignee closes the request.
+It is the responsibility of the Assignee to act and not let PR remain at a stale point.
+
+Whenever possible (unless they are not eligible, or for other reasons known to be unavailable), the Assignee SHOULD be:
+
+1. `@noelmace` for everything related to communication, openness framework, DevX and DevRel in general
+2. `@filipblnt` for everything related to engineering, by default (adding a member of the engineering team as an additional Reviewer whenever needed)
+3. `@milstan` for everything related to our company's "vision" and high-level concepts (i.e. high-level product ownership)
+4. `@Vuk-BN` for everything related to global executive strategies and finances
+
+When such assignment is not possible, the then the Assignee SHOULD be the first eligible and available person from the above list, in the order from 1. to 4.
+
+### Validation protocol for non template-based documents
+
+>This protocol applies, unless a particular project-related workflow is specified in the project's readme file, when producing other types of documents that MAY also involve more or less normative **choices** and even certain **decisions** (e.g. user stories, specifications, designs, documentation work, code) but that do not use one of the [Decision Framework's specific document templates](https://github.com/blindnet-io/openness-framework/tree/main/DecisionFramework#what-template-will-you-use)
+
+One of the Authors, the **Corresponding Author**, makes a PR.
+They invite Reviewers others then themselves.
+A Reviewer MAY designate themselves.
+
+The **Corresponding Author** assigns a person (other than themselves, which MAY also be a Reviewer, in the same order as for template-based decisions).
+
+The Assignee looks for consensus before merging the PR.
+In clear absence of consensus the Assignee closes the request.
```

Comment:
I'm not sure about this one (and maybe even the same point in the previous section).

In some cases, the assignee might still need to make a choice when some reviewers _slightly_ disagree. We might need to add a definition for a **formal objection** here (as opposed to an informal objection, when a reviewer _slightly_ disagrees, but not enough to block the decision). As always, I take my inspiration from Open Standards Organizations (here, [W3C's Processing of Formal Objections](https://www.w3.org/2017/12/formal-objections.html)):

> **Best efforts to resolve via consensus.**
> 
> The delegate should search for potential consensus positions. When that is not possible, at least the delegate is in a position to clearly articulate the rationale for the decision.
> 
> **Analysis of all arguments raised.**
> 
> Although the decision should be **clear, well explained, and unambiguous**, the decision should nonetheless acknowledge good points raised by the party that is not favoured in the ruling, but explain why they are not decisive. 

However, here, closing a PR doesn't necessarily mean ending the decision process, as the discussion might continue in another issue or PR. So we might need to add a definition _"alternative proposal"_ instead or additionnaly.